packages/cadl-ranch-specs/http/content-type/multipart-formdata/main.tsp

```diff
@@ -0,0 +1,74 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+
+@doc("Test content-type: multipart/form-data")
+@scenarioService("/content-type/multipart-formdata")
+namespace ContentType.MultipartFormData;
+
+@encode("binary")
+scalar binary extends bytes;
```

Comment:
you don't need to do that, this is just a workaround for openapi3, use `bytes` directly